tests/operator/operator.go

```diff
@@ -3422,3 +3370,53 @@ func waitForUpdateCondition(kv *v1.KubeVirt) {
 		),
 	)
 }
+
+func waitForKvWithTimeout(newKv *v1.KubeVirt, timeoutSeconds int) {
```

Comment:
Same, does it make sense to expose this? sounds useful outside of this test.
There's already UpdateKubeVirtConfigValueAndWait which is pretty similar